CMakeLists.txt

```diff
@@ -7511,6 +7511,7 @@ target_include_directories(handshake_verify_peer_options
   PRIVATE ${_gRPC_CARES_INCLUDE_DIR}
   PRIVATE ${_gRPC_GFLAGS_INCLUDE_DIR}
   PRIVATE ${_gRPC_ADDRESS_SORTING_INCLUDE_DIR}
+  PRIVATE ${_gRPC_NANOPB_INCLUDE_DIR}
```

Comment:
I'm confused. CMakeLists.txt is generated from templates/CMakeLists.txt.template  which isn't updated here so sanity tests should be failing, but they seem to be passing.

Reply:
Ok, this is because of broken sanity being fixed by https://github.com/grpc/grpc/pull/15810. Please rebase this PR so that we start off of a good revision.

Reply:
Rebased, thanks!